app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -41,12 +41,41 @@ def translate_exception(err)
   def verify_credentials(auth_type = nil, options = {})
     auth_type ||= 'default'
 
+    raise MiqException::MiqInvalidCredentialsError, "Unsupported auth type: #{auth_type}" unless supports_authentication?(auth_type)
     raise MiqException::MiqHostError, "No credentials defined" if missing_credentials?(auth_type)
 
     options[:auth_type] = auth_type
+    case auth_type.to_s
+    when 'default' then verify_api_credentials(options)
+    when 'amqp'    then verify_amqp_credentials(options)
+    end
+  end
+
+  def event_monitor_options
+    @event_monitor_options ||= begin
+      url = ''
+      amqp = connection_configuration_by_role('amqp')
+      if (endpoint = amqp.try(:endpoint))
+        url = "#{endpoint.hostname}:#{endpoint.port}"
+      end
+
+      if (authentication = amqp.try(:authentication))
+        url = "#{authentication.userid}:#{authentication.password}@#{url}"
+      end
+
+      {
+        :ems                       => self,
+        :url                       => url,
+        :sasl_allow_insecure_mechs => true, # Only plain (insecure) mechanism currently supported
```

Comment:
Now that we've introduced settings.yaml, we could probably also move `:sasl_allow_insecure_mechs` decision in there.

Reply:
Insecure will come from the AMQP settings from the UI (you probably remember the "new provider" where you have an option for secure/not secure).

The reason to have this hardcoded is simply because Nuage supposedly only uses amqp and not amqps - thus, for now even the UI will not allow ssl.

Reply:
Ah, yes, you are right. Thanks for the explanation!